README.md

```diff
@@ -73,6 +73,9 @@ NCBI interaction scripts that are useful for fetching public metadata and sequen
 
 - [fetch-from-ncbi-entrez](fetch-from-ncbi-entrez) - Fetch metadata and nucleotide sequences from [NCBI Entrez](https://www.ncbi.nlm.nih.gov/books/NBK25501/) and output to a GenBank file.
   Useful for pathogens with metadata and annotations in custom fields that are not part of the standard [NCBI Virus](https://www.ncbi.nlm.nih.gov/labs/virus/vssi/) or [NCBI Datasets](https://www.ncbi.nlm.nih.gov/datasets/) outputs.
+- [fetch-from-ncbi-virus](fetch-from-ncbi-virus) - Fetch metadata and nucleotide sequences from [NCBI Virus](https://www.ncbi.nlm.nih.gov/labs/virus/vssi/#/) and output NDJSON records to stdout.
```

Comment:
Do you know of a data dictionary for all the csv/ndjson fields returned? The "all-fields" example is amazing, but it would be great if we had a bit more information about what they represent - in case this exists, maybe it doesn't.

Reply:
There's not really any documentation since this is not an official API. The [NCBI Virus help page](https://www.ncbi.nlm.nih.gov/labs/virus/vssi/docs/help/#compare-name-results-in-table) is the closest I can find, but that uses the "pretty" field names that they display on the web UI.